persistent-data-provider/keystone.ts

```diff
@@ -3,7 +3,7 @@ import { statelessSessions, withItemData } from '@keystone-next/keystone/session
 
 import { lists } from './schema';
 
-const sessionSecret = process.env.SESSION_SECRET;
+let sessionSecret = process.env.SESSION_SECRET;
```

Comment:
```suggestion
const sessionSecret = process.env.SESSION_SECRET;
```

Reply:
`sessionSecret` is mutated on line 12, so this will cause an error.